rclcpp/include/rclcpp/client.hpp

```diff
@@ -67,7 +67,8 @@ class ClientBase
 
   virtual std::shared_ptr<void> create_response() = 0;
   virtual std::shared_ptr<void> create_request_header() = 0;
-  virtual void handle_response(std::shared_ptr<void> &response, std::shared_ptr<void> &req_id) = 0;
+  virtual void handle_response(std::shared_ptr<void> &request_header,
+                               std::shared_ptr<void> &response) = 0;
```

Comment:
I'm neutral about this, though. But I'd be more than happy if there's a code style (whatever it is) and we all follow it.